samples/bluetooth/tmap_central/src/cap_initiator.c

```diff
@@ -21,20 +21,20 @@
 static struct k_work_delayable audio_send_work;
 
 static struct bt_cap_stream unicast_streams[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT +
-										CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT];
+					CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT];
 static struct bt_bap_ep *unicast_sink_eps[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT];
 static struct bt_bap_ep *unicast_source_eps[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT];
 
 NET_BUF_POOL_FIXED_DEFINE(tx_pool, CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT,
-						CONFIG_BT_ISO_TX_MTU + BT_ISO_CHAN_SEND_RESERVE, 8, NULL);
+			CONFIG_BT_ISO_TX_MTU + BT_ISO_CHAN_SEND_RESERVE, 8, NULL);
```

Comment:
There still seems to be some general issues with the indentation :) Tabs are 8 spaces, and code should be indented to match previous lines (e.g. by `(`)